src/sql/platform/accounts/test/common/testAccountManagementService.ts

```diff
@@ -35,7 +35,7 @@ export class TestAccountManagementService implements IAccountManagementService {
 		return undefined;
 	}
 
-	copyUserCodeAndOpenBrowser(userCode: string, uri: string): void {
+	copyUserCodeAndOpenBrowser(userCode: string, uri: string): Promise<void> {
 		return undefined;
```

Comment:
Does this compile? It should be throwing an error I would expect since it's not listed as async (also if it's returning void it should just `return;`

Reply:
Yes, it compiled.
![image](https://user-images.githubusercontent.com/87730006/143992749-870559cd-b4d2-428b-85d6-9ac719bd408c.png)

And the 1 problem that appears is due to a unknown compiler option in tsconfig.json.
![image](https://user-images.githubusercontent.com/87730006/143992878-809768cb-1b5e-40e8-a343-292be96aa032.png)

You're definitely right that it should be throwing an error because the TS playground detects an error:
![image](https://user-images.githubusercontent.com/87730006/143993164-e3c413c0-25b7-4587-b0ef-cbc35e484a72.png)

Reply:
Oh yeah, this is because we don't compile in strict mode. TS Playground has that enabled by default - if you make the return type `Promise<void> | undefined` then it'll be happy.

If you could fix it as below that'd be good cleanup

```
copyUserCodeAndOpenBrowser(userCode: string, uri: string): Promise<void> {
    return Promise.resolve();
}
```